geth/api/api.go

```diff
@@ -195,29 +190,17 @@ func (api *StatusAPI) JailBaseJS(js string) {
 	api.b.jailManager.BaseJS(js)
 }
 
-// TODO(oskarth): API package this stuff
+// Notify and send message.
 func (api *StatusAPI) Notify(token string) string {
 	log.Debug("Notify", "token", token)
 
-	var NP fcm.NotificationPayload
-	NP.Title = "Status - new message"
-	NP.Body = "ping"
-
 	// TODO(oskarth): Experiment with this
-	data := map[string]string{
+	msg := map[string]string{
 		"msg": "Hello World1",
 		"sum": "Happy Day",
 	}
 
-	ids := []string{
-		token,
-	}
-
-	c := fcm.NewFcmClient(serverKey)
-	c.NewFcmRegIdsMsg(ids, data)
-	c.SetNotificationPayload(&NP)
-
-	_, err := c.Send()
+	err := api.b.notification.Notify(msg, token)
 	if err != nil {
 		log.Error("Notify failed:", err)
```

Comment:
Let's return an error to the caller?

Reply:
We can return an error but this is an epi change. Is it OK for refactoring issue?

Reply:
It should have been done at the very beginning.
@oskarth?

Reply:
I didn't notice that. I guess there is no point in returning the same value (`token`) which was passed an argument.

Reply:
Feel free to change this to whatever makes sense, it was purely there as I spiked things out for debugging.

Reply:
Ok, i`m going to return {Status: bool, Error: string}.